src/main/java/com/binance/dex/api/client/domain/stake/beaconchain/BeaconChainDelegation.java

```diff
@@ -0,0 +1,46 @@
+package com.binance.dex.api.client.domain.stake.beaconchain;
+
+import com.binance.dex.api.client.domain.stake.sidechain.Delegation;
+import com.binance.dex.api.client.domain.stake.sidechain.SideChainDelegation;
+import com.binance.dex.api.client.encoding.message.Token;
+
+/**
+ * @author Francis.Liu
+ **/
+public class BeaconChainDelegation {
+
+    private Delegation delegation;
+
+    private Token balance;
+
+    public BeaconChainDelegation() {
+    }
+
+    public BeaconChainDelegation(Delegation delegation, Token balance) {
+        this.delegation = delegation;
+        this.balance = balance;
+    }
+
+    public static BeaconChainDelegation createBySideChainDelegation(SideChainDelegation sideChainDelegation) {
```

Comment:
It's weird to create beacon chain delegation by sidechain delegation.
It seems the reason is to reuse the sidechainDelegation related code.
Is it possible to refactor the code with a single `Delegation` class used by both beacon chain and sidechain, while remaining backward compatibility?

Reply:
Common class names belong to side chain and beacon chain have been refactored.